mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -357,6 +369,59 @@ properties:
                       required: true
                       description: |
                         Name of a BigQuery field to be returned with the findings.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'hybridOptions'
```

Comment:
In general, we'd prevent users from specifying empty blocks, but it seems it does not make sense to mark any of its subfields as required in case. Plus, we have test cover the case `hybrid_options {}`, so I assume it should be fine.

Reply:
This was my thought exactly. None of the fields are technically required (the API accepts an empty body), and I couldn't come up with a field or set of fields that seemed required for practical usage. I also didn't want to diverge from the API validations.

It's a little weird to allow `{}`, but it seemed like the best option at the moment.